buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -530,6 +536,40 @@ private void copyExtraConfigFiles() {
         });
     }
 
+    /**
+     * Copies extra jars to the `/lib` directory.
+     * //TODO: Remove this when system modules are available
+     */
+    private void copyExtraJars() {
```

Comment:
We prefer not to couple testclusters with random project configuration like an `extraJars` configuration. 
Would be better to have extra jars work exactly  as extra config files so these jars could be   passed in externally. 

A more generic way would be to add the possibility to add hooks right before the task starts.

Reply:
I moved this outside as suggested, similar to how extraConfigurationFiles work